package.json

```diff
@@ -11,7 +11,6 @@
     "start": "npx nodemon index"
   },
   "dependencies": {
-    "cors": "^2.8.5",
```

Comment:
we might want to leave this in for local testing

Reply:
you'll already have it installed so won't be a problem